.dev/compose.prodtest.yml

```diff
@@ -0,0 +1,40 @@
+# onyl runs in combination with compose.backbone.yml
```

Comment:
```suggestion
# only runs in combination with compose.backbone.yml
```

also include should be used here. https://docs.docker.com/compose/how-tos/multiple-compose-files/include/